src/main.ts

```diff
@@ -1,4 +1,13 @@
 import session from "express-session";
 import express from "express";
 
-console.log("HELLO WORLD")
\ No newline at end of file
+function insecureFunction() {
+    const sessionMiddleware = session({ 
```

Comment:
Not a suggestion, and then:

```suggestion
    const sessionMiddleware = session({ vbnm,
```